src/Main.hs

```diff
@@ -159,9 +159,9 @@ bool = maybeReader (f . map toLower) where
   f _ = Nothing
 
 options :: Parser Options
-options = Options . NE.fromList
-  <$> some (argument str (metavar "FILES"
-    <> help "Solidity files to analyze"))
+options = 
+  (Options . NE.fromList <$> some (argument str (metavar "FILES"
+    <> help "Solidity files to analyze")))
```

Comment:
```suggestion
options = Options . NE.fromList
  <$> some (argument str (metavar "FILES"
    <> help "Solidity files to analyze"))
```

diff noise I think? 😄

Reply:
My local hlint was complaining about it so I had to fix it 😅